packaging/linux/common/khiops-env.in

```diff
@@ -2,7 +2,7 @@
 
 help()
 {
-    echo "Usage: khiops-env [-h] [--env]"
+    echo "Usage: khiops-env [-h] [--help] [--env]"
```

Comment:
I think it should be `[-h, --help]` or something like this, because `-h` and `--help` are not orthogonal.

Reply:
done